api.planx.uk/modules/send/bops/bops.test.ts

```diff
@@ -259,9 +261,11 @@ describe(`sending an application to BOPS v2`, () => {
       .post("/bops-v2/unsupported-team")
       .set({ Authorization: process.env.HASURA_PLANX_API_KEY! })
       .send({ payload: { sessionId: "123" } })
-      .expect(400)
+      .expect(500)
       .then((res) => {
-        expect(res.body.error).toMatch(/not enabled for this local authority/);
```

Comment:
when we get around to reading Pay tokens from integrations - we'll need to remember that I think we do a bit of frontend error message string matching based on this old language to show an error on the Pay component - but shouldn't effect submission destinations as these just log to Hasura !